src/utils/stringCollector.ts

```diff
@@ -0,0 +1,26 @@
+export default class StringCollector {
```

Comment:
some description with example would be nice

Reply:
String collector removed from the request. Will be added later with tests